packages/vertica-nodejs/lib/connection-parameters.js

```diff
@@ -114,7 +114,15 @@ class ConnectionParameters {
     this.backup_server_node = parseBackupServerNodes(val('backup_server_node', config))
     this.client_label = val('client_label', config, false)
     this.workload = val('workload', config, false)
+
+    // client auditing information
     this.client_os_hostname = os.hostname()
+    this.client_type = "Node.js Driver"
+    this.client_version = "1.1.1"
```

Comment:
We are going to have to remember to change this every time we update the version.   Is there any way to base it on the same definition?   Alternatively, can we add a comment somewhere that will make it less likely this will be forgotten in the future?

Reply:
The package.json file is where the definitive version lives. It can't, for example, reference an outside file to get the version number. I will add in a comment so that it doesn't get forgotten

Reply:
Okay well that isn't allowed either. I've had to get creative, but now our test should handle this for us. I'll add a comment there explaining.